docs/excel/excel-add-ins-ranges-precedents-dependents.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Work with formula precedents and dependents using the Excel JavaScript API
+description: 'Learn how to use the Excel JavaScript API to retrieve formula precedents and dependents.' 
+ms.date: 05/28/2021
+ms.prod: excel
+localization_priority: Normal
+---
+
+# Get formula precedents and dependents using the Excel JavaScript API
+
+This article provides code samples that retrieve formula precedents and formula dependents using the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see [Range Object (JavaScript API for Excel)](/javascript/api/excel/excel.range).
+
+An Excel formula often refers to other cells. When a cell provides data to a formula, it is known as a formula "precedent". When a cell contains a formula that refers to other cells, it is known as a formula "dependent". To learn more about Excel features related to relationships between cells, see [Display the relationships between formulas and cells](https://support.microsoft.com/office/display-the-relationships-between-formulas-and-cells-a59bef2b-3701-46bf-8ff1-d3518771d507).
+
+## Get formula precedents
```

Comment:
Since the described method only works on the direct precedents, consider including that in the header.